TODO.md

```
@@ -1,11 +1,19 @@
 ## Currently investigating:
 - (none)
 
-## Researched / Ready for planning:
+## Researched / Ready for implementation:
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)